cfgov/v1/preprocessed/css/main.less

```diff
@@ -66,6 +66,7 @@
 @import (less) "breadcrumbs.less";
 @import (less) "mobile-carousel.less";
 @import (less) "share.less";
+@import (less) "molecules/image-text-50-50.less";
```

Comment:
Move to molecule section (see https://github.com/cfpb/cfgov-refresh/pull/1056/files#diff-ecf3e492d09421b2c6b6a02a5111d892R58)